pydantic/_internal/_generate_schema.py

```diff
@@ -991,7 +991,7 @@ def _pattern_schema(self, pattern_type: Any) -> core_schema.CoreSchema:
                 _validators.pattern_bytes_validator, serialization=ser, metadata=metadata
             )
         else:
-            raise PydanticSchemaGenerationError(f'Unable to generate pydantic-core schema for {pattern_type!r}.')
+            raise PydanticSchemaGenerationError(f"Unable to generate pydantic-core schema for '{pattern_type!r}'.")
```

Comment:
you don't need the quotes here, you already have the `!r`.

Reply:
Thank you! That was before you've pointed me to this the previous time.